src/v3/src/util/settingsUtils.ts

```diff
@@ -246,3 +248,32 @@ export const isOauth2Enabled = (widgetProps: WidgetProps): boolean => {
   }
   return typeof clientId !== 'undefined' && authScheme?.toLowerCase() === 'oauth2';
 };
+
+export const getPageTitle = (
+  widgetProps: WidgetProps,
+  formTitle: string,
+  idxTransaction?: IdxTransaction,
+): string | undefined => {
+  const { brandName, features: { setPageTitle } = {} } = widgetProps;
+
+  // When setPageTitle option is 'undefined', default will be to set title based on page header
+  // When setPageTitle option is 'true', set title based on page header
+  if ((typeof setPageTitle === 'boolean' && setPageTitle === true)
```

Comment:
Wondering if a `switch (typeof setPageTitle)` would be more appropriate or clearer